src/unity/python/turicreate/cython/cy_flexible_type.pyx

```diff
@@ -287,14 +287,10 @@ timedelta_type = datetime.timedelta
 cdef object _image_type
 cdef bint have_imagetype
 
-cdef bint HAS_NUMPY = False
+cdef bint HAS_NUMPY = True
```

Comment:
Should we maybe just remove this variable entirely?

Reply:
Probably. That can come in a later change (feel free to do some cleanup here if you'd like). I just merged this to unblock the PR builds now that it's passing.